tst/EnergyPlus/unit/ConvectionCoefficients.unit.cc

```diff
@@ -1377,4 +1377,509 @@ TEST_F(ConvectionCoefficientsFixture, ConvectionCoefficientsTest_HConvInDependen
     ConvectionCoefficient = DataHeatBalance::HConvIn(1);
 
     EXPECT_EQ(ConvectionCoefficient, ExpectedCoefficient);
-}
\ No newline at end of file
+}
+
+TEST_F(EnergyPlusFixture, AdaptiveModelSelections_ProperConstruction)
```

Comment:
If only all unit tests were this straightforward.

Reply:
Thanks! Ill try to keep up that trend.